Realm/Tests/RLMTestCase.m

```diff
@@ -198,11 +198,15 @@ - (void)waitForNotification:(NSString *)expectedNote realm:(RLMRealm *)realm blo
     [token stop];
 }
 
-- (void)dispatchAsync:(dispatch_block_t)block {
+- (dispatch_queue_t)bgQueue {
     if (!_bgQueue) {
         _bgQueue = dispatch_queue_create("test background queue", 0);
     }
-    dispatch_async(_bgQueue, ^{
+    return _bgQueue;
+}
+
+- (void)dispatchAsync:(dispatch_block_t)block {
```

Comment:
Never mind, this is relevant since it fixes a potential bug in `dispatchAsyncAndWait:`, which I use in the tests. If `dispatchAsync:` is not called before `dispatchAsyncAndWait:`, then `_bgQueue` will not exist. This fixes that by making both call the property to lazily create the queue.